src/routes/home.js

```diff
@@ -16,8 +16,8 @@ const event = calendar => ({
   badge: "Termin",
   title: calendar[0].name,
   text: calendar[0].date,
-  link: calendar[0].link,
-  image: calendar[0].cover.source
+  image: calendar[0].cover.source,
+  tag: calendar[0].tag
```

Comment:
Maybe I'm wrong, but I think the name `tag` shouldn't matter in this case.

Reply:
Renamed it do id so it's easier to understand